### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Left to do
- [x] Implement Filter slider on mobile
- [x] Profile 
- [x] Proposal detail

### CODE REVIEW

File: src/components/Feed/ProposalHeader.js [hunk 1]
```diff
@@ -13,11 +21,13 @@ function ProposalHeader({ proposal, onSelectProposal }) {
...
     >
       <ProposalCreator proposal={proposal} />
-      <ContextMenu>
-        <ContextMenuItem onClick={onSelectProposal}>
-          <IconInfo /> Details
-        </ContextMenuItem>
-      </ContextMenu>
+      {layoutName !== 'small' && (
+        <ContextMenu>
+          <ContextMenuItem onClick={onSelectProposal}>
```
Comment (rperez89) [inline src/components/Feed/ProposalHeader.js:26]: wondering if we even need those menu items only for the details

File: src/components/Feed/ProposalsList.js [hunk 2]
```diff
@@ -1,34 +1,72 @@
 import React, { useRef } from 'react'
-import { Button, GU } from '@1hive/1hive-ui'
+import {
+  BIG_RADIUS,
+  Button,
+  GU,
+  useTheme,
```
Comment (rperez89) [inline src/components/Feed/ProposalsList.js:6]: Lets alphabetize this import

File: src/components/Filters/Filters.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  } = props
+
+  return (
+    <div
+      css={`
+        min-width: 270px;
+        height: fit-content;
+        margin-top: ${3 * GU}px;
+        margin-right: ${8 * GU}px;
+
```
Comment (rperez89) [inline src/components/Filters/Filters.js:42]: blank space

File: src/components/Layout.js [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
+function CustomLayout({ children }) {
+  const { width: vw } = useViewport()
   return (
     <Layout
       breakpoints={BREAKPOINTS}
-      paddingBottom={paddingBottom || 0}
+      parentWidth={vw}
+      paddingBottom={0}
       css={`
+        ${vw < BREAKPOINTS.large && `width: auto;`}
```
Comment (rperez89) [inline src/components/Layout.js:13]: i think here we might be adding `false` to the css because of the  `&&`  operator

**Review by rperez89 — COMMENTED** [review]
LGTM!

File: src/components/Layout.js (hunk 4 above)
Comment (fabriziovigevani) [reply src/components/Layout.js:13]: In this case it works the same as conditional rendering of react components so we don't need to worry about that.

File: src/components/Feed/ProposalHeader.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/Feed/ProposalHeader.js:26]: As discussed offline, the menu item is needed. We cannot do the proposal detail link be the whole card as we also have action buttons on it.

**Pull request merged**